.storybook/blocks/ComponentDetails.jsx

```diff
@@ -356,7 +356,7 @@ export const ResourceLinkContent = ({ heading, alt, logo, href }) => {
  * @param {string} rootClassName - a component's default rootClass arg
  * @returns {string}
  */
-export const ResourceListDetails = ({ packageName, spectrumData = [], rootClassName }) => {
+export const ResourceListDetails = ({ packageName, spectrumData = [], rootClassName, status }) => {
```

Comment:
I can't think of a way to avoid passing another arg here either, now that I know there are other components that don't have `spectrumData`... so I think this approach works fine! Any objection to changing the name from `status` to `isDeprecated`? Since `status` is a boolean that is only true for deprecated components?

Reply:
sure! Just pushed up that change, along with removing that `skipBorder`. https://github.com/adobe/spectrum-css/pull/3129/commits/5376ff8d74ad583be26da8f931ee6d04a1b0a06c